css/cssom/getComputedStyle-pseudo.html

```diff
@@ -150,4 +158,19 @@
     getComputedStyle(div, ":totallynotapseudo").length == 0,
     "Should return an empty style for unknown pseudo-elements starting with colon");
 }, "Unknown pseudo-elements");
+test(() => {
+  let li = document.querySelector('li');
+  let markerStyle = getComputedStyle(li, ':marker');
```

Comment:
Why is there a single colon, not a double?

Is the intention of the test to make "::marker" a valid query but make ":marker" and "marker" invalid ones?

Reply:
This should _not_ return a style, `::marker` is not one of the [CSS2 pseudo-elements](https://drafts.csswg.org/selectors-4/#pseudo-element-syntax) (`:before/:after/:first-line/:first-letter`), so it should be treated as any other invalid pseudo-element would.

Reply:
This is a typo from me. I meant to write `::marker` and yes `:marker` and `marker` are invalid.

Reply:
I fixed my typos and added two tests. The goal here is to make sure that a known `pseudoId` such as `marker` in `marker` or `:marker` is not considered valid for WebKit and Blink.  I hope it clarifies the intent.